crates/bevy_xr/src/actions.rs

```diff
@@ -21,6 +21,7 @@ impl<A: Action> Plugin for ActionPlugin<A> {
     }
 }
 
+#[derive(Debug)]
```

Comment:
Any reason not to implement eq, partial eq, hash, clone, and copy? I would expect almost every public type to have these derived. Except copy, and maybe clone.

Reply:
I just wanted debug so I could easily print it.